dot-gen/src/main/java/com/wipro/ats/bdre/ldg/GetDotForTable.java

```diff
@@ -0,0 +1,132 @@
+        LineageMain lineageMain = new LineageMain();
+        //src table name is provided by user
+        String srcTableName = args[1];
+        logger.info("Generating DOT for the given Column Name: " + args[0]);
+        String targetTableName = null;
+        StringBuffer relationDot = new StringBuffer("");
+        getLineageNodeByColName = new GetLineageNodeByColName();
+        //for each relation a new node is created in LN table, so get all those
+        List<LineageNode> srcTableNodeList = getLineageNodeByColName.getTableDotFromTableName(srcTableName);
+        List<LineageNode> lineageNodeList = new ArrayList<LineageNode>();
+        int counter =0;
+        //now get all those col nodes corresponding to the tables created for each relation
+        for(LineageNode lnodes:srcTableNodeList) {
+            logger.info("----------------------------------src table" + lnodes.getDisplayName());
+             lineageNodeList.add(getDotForTable.getLineageNodeNodeId(args[0], lnodes));
+            logger.info("----------------------------------col node" + lineageNodeList.get(counter++).getDisplayName());
+        }
+        StringBuffer targetTableNode = new StringBuffer("");
+
+        //compare the list of Nodes got from LN table with LR table to get the required node
+        for (LineageNode lineageNode : lineageNodeList) {
+            //this only checks the case where the node is src
+            logger.info("------------------------inside");
+            lineageRelationList = getNodeIdForLineageRelation.execute(lineageNode.getNodeId());
+            for(LineageRelation lineageRelation:lineageRelationList) {
+            //if (lineageRelationList.size() != 0) {
+                logger.info("------------------------inside src check");
+                relationDot.append("\n" + lineageRelation.getDotString() + "\n");                                            //search for target table name in the DOT string
+                logger.info("------------------" + relationDot);
+                targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeByTargetNodeId(), srcTableNodeList.get(0)) + "\n");
+                logger.info("-------------------------" + targetTableNode);
+                if(targetTableNode.equals("same-nodes")) {
+                    targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeBySrcNodeId(), srcTableNodeList.get(0)) + "\n");
+                }
+                logger.info("------------------------------relation DOT: \n" + relationDot);
+             }
+
+            //this should check whether the nodes are target -- ??
+            lineageRelationListForTargetNodes = getNodeIdForLineageRelation.getWhenTargetNode(lineageNode.getNodeId());
+            for(LineageRelation lineageRelation:lineageRelationListForTargetNodes) {
+                //if (lineageRelationList.size() != 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)